torch/csrc/jit/python_compiled_function.cpp

```diff
@@ -102,23 +100,25 @@ struct CompiledFunction {
       return true;
     }
 
-    variable_list run(const variable_list& in_vars) {
+    variable_list run(variable_list inputs) {
+      auto & captured_vars = fn_.captured_vars_;
       JIT_ASSERT(is_ready_);
       AutoNoGIL _gil_guard;
       if (closure_) {
         auto fn = closure_->construct();
-        return (*fn)(in_vars);
+        return (*fn)(inputs);
       } else {
         auto fn = factory_->construct();
         fn->willReleaseVariables(); // forward pass is never reused, so it is safe to release anything it can
-        return fn->apply(in_vars);
+        return fn->apply(inputs);
       }
     }
 
-    PyObject* add_trace(PyObject *args, const variable_list& in_vars) {
+    PyObject* add_trace(PyObject *args, variable_list inputs) {
       JIT_ASSERT(!is_ready_);
       // Start tracing
-      auto trace = tracer::enter(fmap<TraceInput>(in_vars), is_volatile_ ? 1 : (fn_.nderivs_ + 1));
+      auto & captured_vars = fn_.captured_vars_;
```

Comment:
used?

Reply:
It's not. I already fixed it in the other PR, but I can port the changes here if you'd prefer to. Let me know what you think

Reply:
ok, don't need to fix it then.